kuttl-tests/kind-test.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+readonly script_directory="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+readonly project_directory="$(readlink -f "${script_directory}/..")"
+
+# shellcheck source=../metadata.sh
+source "${project_directory}/metadata.sh"
+
+KUBECTL_VERSION=1.18.4
+KUTTL_VERSION=0.5.0
+KUBECTL_KUDO_VERSION=${DS_KUDO_VERSION#v}
+KIND_VERSION=0.8.1
+
+ARTIFACTS=kuttl-dist
+
+OS=$(uname -s | tr '[:upper:]' '[:lower:]')
+KUDO_MACHINE=$(uname -m)
+MACHINE=$(uname -m)
+if [ "${MACHINE}" == "x86_64" ]; then
+  MACHINE=amd64
+fi
+
+mkdir -p bin/
```

Comment:
Please make the script not silently depend on correct $PWD. Prepend, or worst case - `cd` into `$script_directory`.

Reply:
I agree, but on the other hand that's true for many of the scripts in this project. A correct $PWD is expected. I'd love to clean this up not only here but in the other scripts as well. As part of a separate PR. WDYT?

Reply:
Whatever works for you. Personally I'd improve the script while writing it. Too often I'd find that such future cleanup never actually happens, because there are more important things to do :-)